neuroquery/img_utils.py

```diff
@@ -36,12 +36,17 @@ def coords_to_voxels(coords, ref_img=None):
 
 
 def coords_to_peaks_img(coords, mask_img):
+    peaks = coords_to_peaks_arr(coords, mask_img)
+    peaks_img = image.new_img_like(mask_img, peaks)
+    return peaks_img
+
+
+def coords_to_peaks_arr(coords, mask_img):
```

Comment:
nitpick: I think coords_to_peaks_array would be clearer

Reply:
Yes, I'll change that!